### PR TITLE
Add a comment for how the Dockerfile path is resolved.

### PR DESCRIPTION
**Pull request by Alexhuszagh**

### CODE REVIEW

**Comment by Alexhuszagh** [comment]
I'm not sure if this is actually required, but it's useful information on how the path to the Dockerfile is resolved. The context is already documented in `/src/docker/shared.rs`:

https://github.com/cross-rs/cross/blob/ab99baf7f4c39403a0b033cef95d798bd0b9a940/src/docker/shared.rs#L422-L427

File: src/docker/custom.rs [hunk 1]
```diff
@@ -128,6 +128,10 @@ impl<'a> Dockerfile<'a> {
             }
         }
 
+        // note that this is always relative to the PWD: if we have `/Dockerfile`
+        // (relative to the workspace root), then running a build in `/src`
+        // requires the Dockerfile path to be specified as `../Dockerfile`,
+        // and running from `/..` requires `${PACKAGE_NAME}/Dockerfile`.
         docker_build.args(["--file".into(), path]);
```
Comment (Emilgardis) [inline src/docker/custom.rs:135]: having issues parsing this, maybe

```suggestion
        // note that this is always relative to the PWD: if we have `$root/Dockerfile`
        // (relative to the workspace root), then running a build
        // `PWD=$root/src/ cross build`
        // would require the Dockerfile path to be specified as `../Dockerfile`.
        docker_build.args(["--file".into(), path]);
```

Comment (Alexhuszagh) [reply src/docker/custom.rs:135]: If we use this with the `$root`, I think we can change it to `$workspace_root` and then remove `(relative to the workspace root)`.

**Review by Emilgardis — APPROVED** [review]
better than nothing

bors r+

**Pull request merged**